console/src/main/java/org/jline/console/impl/DefaultPrinter.java

```diff
@@ -635,14 +632,15 @@ private AttributedString highlightValue(Map<String, Object> options, String colu
             } else {
                 out = new AttributedString(columnValue(objectToString(options, raw)));
             }
-       }
-       if ((simpleObject(raw) || raw == null) && (hv.containsKey("*") || highlightValue.containsKey("*"))
+        }
+        if ((simpleObject(raw) || raw == null) && (hv.containsKey("*") || highlightValue.containsKey("*"))
```

Comment:
Unwanted space for indentation

Reply:
Why unwanted? Previously it was 7 spaces, which is odd.
I fixed it to be 8.